engine/resources/exec.go

```diff
@@ -54,7 +55,7 @@ type ExecRes struct {
 	// only be used when a Shell is *not* specified. The advantage of this
 	// is that you don't have to worry about escape characters.
 	Args []string `yaml:"args"`
-	// Cmd is the dir to run the command in. If empty, then this will use
+	// Cwd is the dir to run the command in. If empty, then this will use
```

Comment:
This is a good catch, but technically it's unrelated to this commit and so it should be in a separate commit. If the commit as-is was perfect except for this, I'd probably make an exception and just merge, but it's good practice for you to learn how to "split a commit" (google or ddg that phrase) so let's do it here! (And remember to fix commit message!)